source/game.programme.newsevent.bmx

```diff
@@ -1166,7 +1166,7 @@ Type TGameModifierNews_ModifyAvailability Extends TGameModifierBase
 	Method Init:TGameModifierNews_ModifyAvailability(data:TData, extra:TData=Null)
 		If Not data Then Return Null
 
-		If data And data.GetString("time") Then self.data = data.copy()
+		InitTimeDataIfPresent(data)
 
 		newsGUID = data.GetString("news", "")
```

Comment:
Scheinbar tanzt der News-ModifyAvailability-Modifier ein wenig aus der Reihe zu tanzen - alle anderen haben "guid" als Key, nur hier "news".
Laesst sich das vereinfachen oder muesste man durch die Speicherstaende gehen und da reparieren?

Reply:
Das ist historisch schon immer so und insb. bei newsChoice (genau eine von x möglichen Nachfolgenachrichten) gibt es dann news1, news2 ...
Also ja, die Nachrichten scheren hier aus. Das in der Datenbank, im Code und in Spielständen gerade zu biegen, wäre nochmal ein ziemlicher Aufwand (und sollte wenn überhaupt außerhalb dieses PR gemacht werden).

Reply:
OK